Documentation/gettingstarted/egress-gateway.rst

```diff
@@ -404,7 +428,7 @@ We can now verify with the client pod that the policy is working correctly:
     [...]
 
 The access log from Nginx should show that the request is coming from the egress
-IP (``192.168.60.100``) rather than one of the nodes in the Kubernetes cluster:
+gateway node IP rather than the one of the node where the pod is running on:
```

Comment:
It's coming from the selected Egress IP. I don't think we should introduce another term ("egress gateway node IP") for it.